### ISSUE TITLE
Can't schedule update

### ISSUE THREAD

**Issue by Azodox** [DevNatan/inventory-framework#169]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### 🐛 Describe the bug

Hi, I think I genuinely don't understand how `scheduleUpdate` works.

Here is my code :
```java
package fr.olten.xmas.clan.view;

import fr.olten.xmas.clan.Clan;
import fr.olten.xmas.ladder.RankingLadder;
import fr.olten.xmas.util.ItemBuilder;
import me.saiintbrisson.minecraft.View;
import me.saiintbrisson.minecraft.ViewContext;
import net.kyori.adventure.text.Component;
import net.kyori.adventure.text.minimessage.MiniMessage;
import org.bukkit.Bukkit;
import org.bukkit.Material;
import org.jetbrains.annotations.NotNull;

public class ClanView extends View {

    private final Clan clan;

    public ClanView(Clan clan) {
        super(5 * 9, "§6§lInformations §8❱ §r" + clan.getName());

        this.clan = clan;

        this.setCancelOnClick(true);
        this.setCancelOnClone(true);
        this.setCancelOnDrag(true);
        this.setCancelOnDrop(true);
        this.setCancelOnMoveOut(true);
        this.setCancelOnPickup(true);
        this.setCancelOnShiftClick(true);

        var fallbackItem = new ItemBuilder(Material.BEDROCK, 1).build();

        this.slot(2, 5, fallbackItem).onRender(render -> {
            if(this.clan.getOwner().equals(render.getPlayer().getUniqueId())){
                render.setItem(new ItemBuilder(Material.TOTEM_OF_UNDYING, 1)
                        .displayName("§e§lVous êtes le propriétaire du clan")
                        .build());
            }
        });

        this.slot(3, 3, fallbackItem).onRender(render -> render.withItem(new ItemBuilder(Material.BOOK, 1)
                .displayName("§b§lMembres")
                .lore(clan.getPlayers().stream().map(player -> "§7" + Bukkit.getOfflinePlayer(player).getName()).toList())
                .build()));
        this.slot(3, 5, fallbackItem).onRender(render -> render.withItem(new ItemBuilder(Material.PAPER, 1)
                .displayName("§6§lInformations")
                .lore(
                        "§7Nom : " + clan.getName(),
                        "§7DisplayName : " + MiniMessage.miniMessage().serialize(clan.displayName()),
                        "§7Couleur : " + clan.color(),
                        "§7Propriétaire : " + Bukkit.getOfflinePlayer(clan.getOwner()).getName()
                )
                .build()));
        this.slot(3, 7, fallbackItem).onRender(render -> render.withItem(new ItemBuilder(Material.LADDER, 1)
                .displayName("§a§lClassement : #" + RankingLadder.getRanking(clan))
                .build()));
        this.slot(5, 4, new ItemBuilder(Material.REDSTONE, 1).displayName("§c§lStatus des options").build());
        this.slot(5, 5, new ItemBuilder(Material.LEGACY_REDSTONE_COMPARATOR, 1).displayName("§c§lGérer les options de clan").build());
        this.scheduleUpdate(20);
    }

    @Override
    protected void onRender(@NotNull ViewContext context) {
        context.getPlayer().sendMessage(Component.text("render"));
    }

    @Override
    protected void onUpdate(@NotNull ViewContext context) {
        context.getPlayer().sendMessage(Component.text("update"));
    }
}
```
The message "render" is correctly sent but not "update"
![](https://i.imgur.com/UJWvw3V.png)

### ✔️ Expected behavior

Each seconds, a message "update" is sent to the player.

### 👣 Steps to Reproduce

1. Create a view
2. Call scheduleUpdate method
3. Override onUpdate
4. Try to send a message or something else in this method

### 💻 Platform

- [X] Bukkit
- [ ] Sponge
- [X] Other

### ⭐ Version

1.19

### ✍️ Additional context

(on Paper)

**Comment by DevNatan** [comment]
(ignore this)
Scheduled update pipeline interceptor must intercept RENDER phase not OPEN phase
https://github.com/DevNatan/inventory-framework/blob/8a08101975d51eff1f465ae4cbc0cf6fd3088dc5/bukkit-api/src/main/java/me/saiintbrisson/minecraft/BukkitViewComponentFactory.java#L164

**Comment by DevNatan** [comment]
Planned to v2.5.3

**Comment by DevNatan** [comment]
@Azodox Fixed in #175. It will be released on v2.5.3. You download the new fixed artifact from [**Generate Artifacts**](https://github.com/DevNatan/inventory-framework/actions/runs/2757918360) workflow (scroll down) or try out [`631cdd605d`](https://jitpack.io/#DevNatan/inventory-framework/631cdd605d) library version.
